internal/oidcserver/types.go

```diff
@@ -53,4 +54,5 @@ const (
 	ServerFlowTypeInvalidCert
 	ServerFlowTypeMissingToken
 	ServerFlowTypeMissingCert
+	ServerFlowHealthz
```

Comment:
This is not needed as well as this endpoint is not part of oidc format. This might cause issues.

Reply:
removed.